.changeset/serious-dancers-whisper.md

```diff
@@ -0,0 +1,6 @@
+---
+"nextjs-website": patch
```

Comment:
**suggestion**: remove the `nextjs-website`

there are no changes on that application, so the changeset for that can be removed